README.md

```diff
@@ -23,7 +23,7 @@ Include a minified browser bundle directly in your HTML like so:
 ```html
 <script
   src="https://unpkg.com/algosdk@v2.1.0/dist/browser/algosdk.min.js"
```

Comment:
Need to also bump the version in these snippets
```suggestion
  src="https://unpkg.com/algosdk@v2.2.0/dist/browser/algosdk.min.js"
```

Reply:
Fixed